lib/ansible/modules/remote_management/manageiq/manageiq_provider.py

```diff
@@ -672,6 +672,9 @@ def main():
         argument_spec=argument_spec,
         required_if=[
             ('state', 'present', ['provider'])],
+        required_together=[
+            ['host_default_vnc_port_start','host_default_vnc_port_end']
```

Comment:
> ['host_default_vnc_port_start','host_default_vnc_port_end']

`['host_default_vnc_port_start',<missing space here>'host_default_vnc_port_end']`
should be:
`...nc_port_start', 'host_de...`

Reply:
Thanks @yaacov got there eventually :)